### ISSUE TITLE
Can you provide some Controller to return json object?

### ISSUE THREAD

**Issue by xkcoding** [schemaspy/schemaspy#333]
Can you provide some Controller to return json object? Not only support generate html?

or  Can you support a maven jar in maven center and this jar can provide some methods to get db metadata information conveniently(include table meta , view meta, procedure meta , column meta , table lineage and impact) , I think it will be more easier to integration other project !

BTW thank you for providing this awesome project !

**Comment by npetzall** [comment]
Maven and integration there is issue #157 
As for output formats there is only xml and html. 

So feature request for report in json?

**Comment by xkcoding** [comment]
Can I call methods to return database's information (include table meta , view meta, procedure meta , column meta , table lineage and impact)?

**Comment by npetzall** [comment]
You can wire everything up and get the schemaspy model of the database. But currently it's a bit of a hassle. But it can be done. The idea from me is that this will be handled in #157 since we've talked about splitting schemaspy to application and libraries)

In regards to what information is stored in the model of the database, I'm gonna say that it probably doesn't contain everything. 
I'm always unsure when it comes to lineage but are we talking relationship(Foreign key, parent/child)? If so it will contain that information. 
Impact is it the on delete/update restriction? if so it will contain that information.

**Comment by xkcoding** [comment]
yep , I mean relationship(Foreign key, parent/child) and delete/update restriction.

BTW , #157  when it will be handled ?

**Comment by xkcoding** [comment]
thank you for your time

**Comment by bryancjacobs** [comment]
Currently, I'm parsing all the metadata out of the HTML files.  It's a bit yucky too do that but the tool works really well.

An improvement for me would be the ability to get that JSON dump of metadata.  Then I could simply use that JSON metadata to send to my Apache Atlas Metadata repository for all my databases.

**Comment by npetzall** [comment]
@bryancjacobs you also have the xml output that could be transformed to json.

I sort of have the feeling that it might be missing information. 

There's also a problem with the model, it's modified when analyzed. So xml will look different if html is generated or not with `-nohtml`
